src/org/mozartoz/truffle/nodes/control/AndThenNode.java

```diff
@@ -0,0 +1,23 @@
+package org.mozartoz.truffle.nodes.control;
+
+import org.mozartoz.truffle.nodes.OzNode;
+
+import com.oracle.truffle.api.frame.VirtualFrame;
+import com.oracle.truffle.api.profiles.ConditionProfile;
+
+public class AndThenNode extends OzNode {
+	private @Child OzNode left, right;
```

Comment:
Usually each `@Child` is on its own line, and `private` is not added as it hurts a bit readability (IMHO)